### PR TITLE
Use ObjectEntryIndex for entrance style

### PR DESCRIPTION
**Pull request by Gymnasiast**
Ported from NSF.

### CODE REVIEW

File: src/openrct2/actions/RideSetAppearanceAction.h [hunk 1]
```diff
@@ -29,7 +29,7 @@ DEFINE_GAME_ACTION(RideSetAppearanceAction, GameCommand::SetRideAppearance, Game
 private:
     NetworkRideId_t _rideIndex{ RIDE_ID_NULL };
     RideSetAppearanceType _type{};
-    uint8_t _value{};
+    uint16_t _value{};
```
Comment (ZehMatt) [inline src/openrct2/actions/RideSetAppearanceAction.h:32]: Is this ObjectEntryIndex? Also the type was not changed in the constructor.

Comment (Gymnasiast) [reply src/openrct2/actions/RideSetAppearanceAction.h:32]: _Sometimes_ it’s ObjectEntryIndex. `RideSetAppearanceAction` is used for several different settings.

Comment (ZehMatt) [reply src/openrct2/actions/RideSetAppearanceAction.h:32]: I see, either way you should probably changed uint8 to uint16 of the constructor parameter.

Comment (Gymnasiast) [reply src/openrct2/actions/RideSetAppearanceAction.h:32]: That was an oversight. I have now adjusted them.

**Review by ZehMatt — APPROVED** [review]

**Pull request merged**